projects/angular/src/forms/textarea/STYLES.md

```diff
@@ -0,0 +1,25 @@
+# Textarea
+
+## CSS Properties
+
+| CSS Property Name                            | Description                                        |
+| -------------------------------------------- | -------------------------------------------------- |
+| --clr-forms-textarea-background-color        | Background color for textarea elements.            |
+| --clr-forms-select-caret-hover-color         | Hover color for the caret icon in select elements. |
+| --clr-forms-select-caret-color               | Color for the caret icon in select elements.       |
+| --clr-forms-select-option-color              | Color for the options in select elements.          |
+| --clr-forms-select-multiple-background-color | Background color for multiple select elements.     |
+| --clr-forms-select-multiple-border-color     | Border color for multiple select elements.         |
+| --clr-forms-select-multiple-option-color     | Color for the options in multiple select elements. |
+
+## Classes
+
+| Class Name                                 | Description                                                                |
+| ------------------------------------------ | -------------------------------------------------------------------------- |
+| clr-textarea-wrapper                       | Wrapper class for the textarea element                                     |
+| clr-validate-icon                          | Icon class used for validation in the textarea wrapper                     |
+| clr-textarea                               | Class for the textarea element                                             |
+| clr-error                                  | Class applied to the parent element when there is an error in the textarea |
```

Comment:
I think that's more of a general class and it's used by the textarea, not declared by it.